site/content/entry/yuzu-progress-report-aug-2022/index.md

```diff
@@ -0,0 +1,334 @@
+One feature that was very useful in the early days is loading ELF binaries.
+Instead of dealing with XCI and NSP dumps, back in the early days, devs dumped and extracted the whole file structure of the game, and loaded the ELF binary directly, in a similar way to how on Windows the user opens a .exe file to load a program.
+
+Some years down the line and not only this feature [tarnished out of grace](https://www.youtube.com/watch?v=6R3ouGNcACQ), but it also stopped working.
+Since it’s not a priority at the moment, and it can lead to user confusion, byte[] {{< gh-hovercard "8701" "removed it from the File submenu." >}}
+
+Asian Windows users noticed that the Size column in the game list and the Speed Percent setting displayed *very weird* numbers.
+
+{{< single-title-imgs
+    "Alien numbers"
+    "./bad1.png"
+    "./bad2.png"
+    >}}
+
+Of course this isn’t acceptable, so [Docteh](https://github.com/Docteh) implemented the {{< gh-hovercard "8715" "necessary changes" >}} to workaround this issue under Windows.
+
+{{< imgs
+	"./digits.png| Much better"
+  >}}
+
+Following up with Asian only UI bugs, If the user started yuzu for example in Chinese and then switched to English, the Filter bar at the bottom and the game list columns at the top would remain untranslated, refusing to switch.
+{{< gh-hovercard "8797" "Forcing Qt to retranslate" >}} these objects solves the issue, making Docteh happy.
+
+Docteh also implemented some {{< gh-hovercard "8741" "nice cleanups" >}} to the About dialog.
+
+{{< single-title-imgs-compare
+	" "
+	"./aboutbug.png"
+	"./aboutfix.png"
+>}}
+
+Users complained that the Control profiles drop down list had a too short character limit, not allowing them to be creative.
+german77 stepped up and {{< gh-hovercard "8783" "increased the character limit." >}}
+
+## Input improvements
+
+UI navigation with a controller is relatively recent in yuzu, so there is room for improvement.
+An example reported by users is that if asking for the user profile at boot is enabled, the controller will be able to navigate the options and start the game, but it won’t be able to actually change the profile, or cancel the boot process.
+Newcomer [jonesyUK](https://github.com/jonesyUK) {{< gh-hovercard "8716" "solves both issues" >}} with a bit of Qt magic.
+Thank you!
+
+Some users prefer to depend on ds4windows instead of yuzu’s native support for game controllers.
+While there is a good reason for that, it’s a simple way to have a single app handle XInput emulation, it can also lead to some interesting issues german77 discovered.
+For example, current versions of ds4windows can’t take two sequent vibration commands, producing a (depending on the person you ask) not very comfortable constant vibration.
+To solve this brrrrrracing issue, german77 {{< gh-hovercard "8722" "added a small delay" >}} so the app doesn’t skip the stop signal.
```

Comment:
```suggestion
Some users prefer to depend on [DS4Windows](https://github.com/Ryochan7/DS4Windows) instead of yuzu’s native support for game controllers.
While there is a good reason for that, it’s a simple way to have a single app handle XInput emulation, it can also lead to some interesting issues german77 discovered.
For example, current versions of DS4Windows can’t handle two subsequent vibration commands, producing (depending on the person you ask) an uncomfortable, constant vibration.
To solve this brrrrrracing issue, german77 {{< gh-hovercard "8722" "added a small delay" >}} so the app doesn’t skip the stop signal.
```